src/js/utils/styles.js

```diff
@@ -76,10 +76,6 @@ export const baseStyle = css`
   -ms-text-size-adjust: 100%;
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased;
-
-  * {
```

Comment:
Can you explain a bit more about what removing this fixes? I ask because this still seems to be the recommended best practice of resetting box-sizing, based on https://css-tricks.com/box-sizing/.